### PR TITLE
Reinstate windows dev build/test

### PR DESCRIPTION
**Pull request by ceball**
While I think the plan is for someone full of energy to move holoviz to github actions, in the meantime I battled through our self-inflicted 17 different appveyor and github accounts until I managed to sign in to what I think is the latest combination. I created the holoviz-developers/panel project on appveyor.

Note: I deleted an old appveyor webhook and installed one from the latest account. If this PR is declined or unsuccessful, the new webhook should be deleted at the same time this PR is closed.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
-    CHANS_DEV: "-c pyviz/label/dev -c bokeh -c conda-forge"
+    CHANS_DEV: "-c pyviz/label/dev -c bokeh/label/dev -c conda-forge"
   matrix:
-    - PY: "3.6"
-      CONDA: "C:\\Miniconda36-x64"
+    - PY: "3.7"
+      CONDA: "C:\\Miniconda37-x64"
 
 install:
   - "SET PATH=%CONDA%;%CONDA%\\Scripts;%PATH%"
```
Comment (ceball) [inline .appveyor.yml:9]:
```suggestion
  - "SET PATH=%CONDA%;%CONDA%\\Scripts;%CONDA%\\Library\\bin;%PATH%"
```

**Comment by ceball** [comment]
@philippjfr from a quick glance it looks like I got this to the same stage as travis:

win:

Screenshot from 2020-10-21 13-12-11

linux:

Screenshot from 2020-10-21 13-12-36

**Comment by xavArtley** [comment]
what's the recommended way to deal with this?
I suppose `test_manual.py` is a test which must be run manually
There is several options to deal with such files :
 - First rename files and remove the `test_` prefix (set it as suffix)
- Or in tox.ini add an option `--ignore-glob="*_manual.py"`

**Comment by ceball** [comment]
I've updated to keep up with changes to the badges/appveyor link on master (no idea whether the badge should come from appveyor or shields.io; also not sure why updating the appveyor badge on master separately from this PR...).

**Comment by philippjfr** [comment]
VTK seems to be crashing things, this was kind of a pain to configure on Travis so I wonder if we just skip these for now.

**Comment by xavArtley** [comment]
don't know if it could help but pyvista, to install opengl on appveyor, use this https://github.com/pyvista/gl-ci-helpers/blob/master/appveyor/install_opengl.ps1
used in their azure pipeline:
```
- powershell: |
      Set-StrictMode -Version Latest
      $ErrorActionPreference = "Stop"
      $PSDefaultParameterValues['*:ErrorAction']='Stop'
      git clone --depth 1 git://github.com/pyvista/gl-ci-helpers.git
      powershell gl-ci-helpers/appveyor/install_opengl.ps1
```

**Comment by philippjfr** [comment]
Wow that actually appears to have worked, thanks @xavArtley!

**Comment by philippjfr** [comment]
And thanks @ceball of course! Glad to have appveyor tests again.

**Pull request merged**